nesta_ds_utils/networks/build.py

```diff
@@ -78,3 +123,91 @@ def _cooccurrence_counts(sequences):
     """
     combos = (combinations(sorted(set(sequence)), 2) for sequence in sequences)
     return Counter(chain(*combos))
+
+
+def _frequency_counts(all_tokens: List) -> dict:
+    """counts frequency of all terms in corpus
```

Comment:
I know this is used above, too, but what do you think about using language that is more agnostic to use than 'terms'?